### DISCUSSION TITLE
Global labels

### DISCUSSION THREAD

**Discussion by koopa1338** [JordanKnott/taskcafe#97] [Ideas]
I'm testing this application for my little side projects, and so far I'm liking it. What would make sense especially for managing multiple projects are global labels. Labes as `Bug`, `Feature` or `Documentation` may be part of most projects.

**Comment by JordanKnott** [comment]
Hmm - I do see how this could be useful. One thing I had planned on doing when I implement custom fields is having a global "field library" so that you can easily pull in often used custom fields. Perhaps I can do something similar with labels? Perhaps a global label library that let's you easily add label bundles (a collection of `n` labels).

Another planned feature is project templates which could also be another viable solution.
